eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -300,6 +300,13 @@ object Features {
     val mandatory = 154
   }
 
+  // TODO: @remyers reserve feature bits here: currently reserved here: https://github.com/lightning/bolts/issues/605
+  // TODO: @remyers option_quiesce implementation, to be replaced once quiescence is spec-ed
+  case object QuiescePrototype extends Feature with InitFeature {
+    val rfcName = "option_quiesce_prototype"
+    val mandatory = 158
```

Comment:
Why don't you use the real feature bit from the spec PR? I don't think see a good reason to use a temporary one...

Reply:
The [quiescence spec PR](https://github.com/lightning/bolts/pull/869) uses 34/35, but that is not listed in the [waiting room](https://github.com/lightning/bolts/issues/605). In fact the waiting room uses 34/35 for 'option_inv_gossip'.

Should I still use 34/35 or work this out first on the lightning-dev call and then update the waiting room?

Reply:
otoh - we can assume the waiting room is out of date, so I took your suggestion to use the spec bits.

Fixed in [9fa905a](https://github.com/ACINQ/eclair/pull/2680/commits/9fa905a3e557420b011b8c5079cb48ba74b705a2)

Reply:
Yes, spec PRs always have precedence, you should just ignore the waiting room (which shouldn't use low feature bits in the first place).